_guides/ctcp.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: documentation
+title: Send CTCP commands with The Lounge
+---
+
+[CTCP (Client-to-client protocol)](https://en.wikipedia.org/wiki/Client-to-client_protocol)
+commands are special messages that can be sent to a channel or other clients.
+Servers do not interpret their content, as it is the role of the receiving
+client to do so.
+
+In fact, you might have used a CTCP command without knowing it. In The Lounge
+and most IRC clients, sending `/me says hello world` actually sends the CTCP
+`ACTION` command with `says hello world` as a parameter. It is then interpreted
+by clients to display it in the way you are familiar with.
+
+In The Lounge, you can send a CTCP command by sending messages starting with
+`/ctcp`. If you are sending requests to other users using The Lounge, here are
+the currently implemented ones:
+
+- `/ctcp <nick> version` returns something along the lines of
+  `VERSION thelounge v3.0.0 -- https://thelounge.chat/`
+- `/ctcp <nick> source` returns
+  `SOURCE git+https://github.com/thelounge/thelounge.git`
+- `/ctcp <nick> ping foo` returns `PING foo`
+
+To know more about internal details of how The Lounge implements CTCP responses,
+take a look at
+[the related source code](https://github.com/thelounge/thelounge/blob/master/src/plugins/irc-events/ctcp.js).
+
+The Lounge does not support all CTCP command. In particular, it does not
+currently support
+[DCC (Direct Client-to-Client)](https://en.wikipedia.org/wiki/Direct_Client-to-Client).
+
+A note of warning: when querying other users, know that their clients might
+inform them that you are requesting information about them, in the form of a
+notification or sound.
+
+If you want to learn more about CTCP, [this document](http://modern.ircdocs.horse/ctcp.html) is a great reference, more
```

Comment:
That page links to a RFC (draft) now: https://tools.ietf.org/html/draft-oakley-irc-ctcp-02

Reply:
Good catch. I think https://rawgit.com/DanielOaks/irc-rfcs/master/dist/draft-oakley-irc-ctcp-latest.html is much more reader-friendly and effectively has the same content. Are you okay if I use this one instead?

Reply:
> Rawgit will die soon.

What do you mean?

Reply:
See https://rawgit.com/

Reply:
Ah, right, I didn't catch that. Switching to the IETF link now.